lib/terraforming/resource/db_security_group.rb

```diff
@@ -46,7 +46,17 @@ def ingresses_of(security_group)
       end
 
       def db_security_groups
-        @client.describe_db_security_groups.map(&:db_security_groups).flatten.select { |sg| !ingresses_of(sg).empty? }
+        marker = ""
+        dbsgs = []
+
+        loop do
+          resp = @client.describe_db_security_groups(marker: marker)
+          dbsgs += resp.db_security_groups.select { |sg| !ingresses_of(sg).empty? }
```

Comment:
__[rubocop]__
* Use `reject` instead of inverting `select`.

<!-- Comment by [SideCI](https://sideci.com) -->